Gems/EMotionFX/Code/EMotionFX/Rendering/Common/RenderUtil.h

```diff
@@ -735,7 +720,7 @@ namespace MCommon
          * @param radius The radius of the sphere.
          * @return A pointer to the newly created util sphere mesh.
          */
-        static UtilMesh* CreateSphere(float radius, uint32 numSegments = 8);
+        static UtilMesh* CreateSphere(float radius, uint32 numSegments = 5);
```

Comment:
What's up with changing this default?

Reply:
Good spot, should have commented that in the PR actually. These are used for rendering line-based skeletons and heavily drop performance (due to the really bad way we render these). As they are super small anyway, I decreased the number of segments to trade that with some higher fps. This will all be replaced/removed with the Atom viewport changes - we'll use the aux geom and remove the Util meshes from here entirely. For line-based skeleton, we shouldn't even render spheres, but just scaled points or something similar.